### ISSUE TITLE
Budget for rebranding: £1,200

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/london-money#47]
I want to get some budget for a re-branding exercise to be led by @nogainbar who has already made a start on this with the help of @markwilliamfirth, @NataliaLKB  and others. See https://github.com/foundersandcoders/assets/issues.

Free time only gets us so far and this is feeling somewhat urgent, see e.g. https://github.com/foundersandcoders/nazareth-programme/issues/175

Noga estimates 10 days @ £120/day.

**Comment by iteles** [comment]
@sofer I don't think this an be agreed to without a list of what the expectations are. i.e. a checklist of what needs to be prepared for @nogainbar before she starts and a checkist of what Founders & Coders needs from those 10 days (even if must-haves and nice-to-haves) - 'rebranding' is a bit too broad for budget to be agreed 😊

**Comment by nogainbar** [comment]
here is the list, @iteles  thanx for asking for clarification, I agree that is essential for everyone to understand what this funding is going towards and what are the expectations.

# FAC rebranding

## Must haves:
 1. **Three logo variants:**
- One variation of the current logo
- Two entirely new designs

All logos should work in both colour and monochrome (Dark, light)

2. ### New colour palette:
**Three options to choose from, each composed of primary, accent, black and white :**
* One variation of current palette.
* Two entirely new palettes.

3. ### New font:
**Three options of new font to be used on all online platforms, forms and communications, including a suggested font hierarchy.**
Fonts should be suitable for both web and print.

4. ### Detailed documentation: 
**In the form of a github repo** of how to implement FAC's new branding so that it can be applied easily and consistently throughout all campuses.

## Nice to have:
* Examples of how the new branding can be applied on the current  website, before embarking on a complete redesign of our website.
* Examples of how the new logo can be applied on t-shirts.
* Examples of how the logo can be applied on posters.
* Examples of how the new branding can affect various design decisions like components in web design.
* Presentation including all the "mood boards", mind maps and process behind the decisions that were made for the design. (this one is something I would really try to accomplish in the time given so that the thought process can be brought back to the FAC community when voting on the new branding).

PO for this rebrand would be @sofer,
however the final decision should be put out for a vote by the wider Founders & Coders community and I suggest it can be done by a poll.

Note: since this repo is public, it is important for me to mention that the rate is a 'mate rate' and is not my usual rate, however I am more than happy to do this as it is dear to me and I am hoping it can make a difference to the general marketing and brand of Founders & Coders.
